docs/import/manage_disaster.html

```diff
@@ -103,19 +103,19 @@
     </form>
   </div>
 <hr/>
-  <div id="current-disaster-interaction">
-    <button type="button" id="delete" disabled>Delete Current Disaster</button>
-    <div id="score-asset-selection">
-      <div class="form-title">Select Score Data</div>
-      <div>
+  <div id="current-disaster-interaction" style="width:100%">
```

Comment:
I think it would be nice to keep all our styles in the CSS but if we're time pressed that's okay I can clean it up later.